woocommerce/woocommerce-ios
# Upgrade project to Xcode 14.0 stable recommended settings

### Description
Noticed a couple of warnings while testing the Xcode 14 branch.

### Testing instructions
If CI is green, this is good. But one could checkout this branch and verify the "upgrade project..." warnings are gone.

---
